### PR TITLE
use strong name

### PR DESCRIPTION
**Pull request by LittleLittleCloud**

### CODE REVIEW

File: src/Plotly.NET/Plotly.NET.fsproj [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0; net5.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>Open.snk</AssemblyOriginatorKeyFile>
```
Comment (LittleLittleCloud) [inline src/Plotly.NET/Plotly.NET.fsproj:5]: You will need to [create your own public-private key](https://docs.microsoft.com/en-us/dotnet/standard/assembly/create-public-private-key-pair#create-a-key-pair) and replace "Open.snk" here

Comment (WhiteBlackGoose) [reply src/Plotly.NET/Plotly.NET.fsproj:5]: You can create it yourself without the passphrase and push it to the repo.

**Review by WhiteBlackGoose — CHANGES_REQUESTED** [review]
The problem is that all deps of a package must be strong named too, like [DynamicObj](https://github.com/CSBiology/DynamicObj/blob/main/src/DynamicObj/DynamicObj.fsproj).

Also need to make sure that [Newtonsoft.Json](https://github.com/JamesNK/Newtonsoft.Json) is strong named (can't verify it right now).

**Comment by kMutagene** [comment]
Closed as this is implemented via #298

**Pull request closed without merging**